rbi/core/method.rbi

```diff
@@ -6,4 +6,49 @@ class Method < Object
 
   sig {params(args: T.untyped).returns(T.untyped)}
   def call(*args);end
+
+  sig {params(_: T.untyped).returns(T.untyped)}
+  def <<(_); end
```

Comment:
These argument names show up in error messages when called incorrectly. Would you mind cross-referencing with the ruby docs to see what names they use for these arguments? Or if you can't find any, can you make up something instead of using `_`?